.github/workflows/release.yml

```diff
@@ -0,0 +1,320 @@
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+        compiler: [gnu]
+      fail-fast: true
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install libomp package
+        run: brew install libomp
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pytest tqdm
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Test with pytest
+        run: |
+          env
+          pip install --pre --find-links ./wheels/ "scikit-decide[all]"
+          pytest -v -s tests/autocast
+          pytest -v -s tests/solvers/cpp
+          pytest -v -s tests/solvers/python
+          pytest -v -s tests/scheduling
+
+  test-windows:
+    needs: build
+    strategy:
+      matrix:
+        os: [windows-latest]
+        python-version: [3.7, 3.8]
+        compiler: [gnu]
+      fail-fast: true
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pytest tqdm
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Test with pytest
+        run: |
+          pip install --pre --find-links ./wheels/ "scikit-decide[all]"
+          pytest -v -s tests/autocast
+          pytest -v -s tests/solvers/cpp
+          pytest -v -s tests/solvers/python
+          pytest -v -s tests/scheduling
+
+  upload:
+    needs: [test-unix, test-macos, test-windows]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Get the version
+        id: get_version
+        run: |
+          echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Upload artifacts 📦 to release
+        uses: svenstaro/upload-release-action@v1-release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: wheels/*.whl
+          tag: ${{ steps.get_version.outputs.VERSION }}
+          overwrite: true
+          file_glob: true
+
+  deploy:
+    needs: [upload]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Publish distribution 📦 to PyPI
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
@galleon Which account does it use by default (say when the script is run from Airbus' scikit-decide repository)?

Reply:
It uses `scikit-decide` project on PyPI. The secret can be changed from github admin page. For the time being we are both users on this `scikit-decide` project.